psi4/src/psi4/libmints/electricfield.h

```diff
@@ -108,6 +118,7 @@ class ContractOverDensityFieldFunctor {
     ContractOverDensityFieldFunctor(SharedMatrix field, SharedMatrix D) : pD_(D->pointer()), field_(field->pointer()) {
         if (D->rowdim() != D->coldim()) throw PSIEXCEPTION("Invalid density matrix in ContractOverDensityFieldFunctor");
         if (field->coldim() != 3) throw PSIEXCEPTION("Field matrix must have 3 columns.");
+        field->zero();
```

Comment:
Sorry, I meant to not zero this variable out....as you had it before.

My original comment was comparing this one to the dipole functor: the dipole one you used to zero it out and this one you didn't. So I wanted the dipole one to behave like this one was.

Once this is fixed, someone else can feel free to merge it. I'll be away from a computer for most of the day.

Reply:
Now I'm confused...

Just to clarify: The suggestion is *not* to zero out anything?

Reply:
Right. You weren't before which allowed the caller to do what they wanted (add to an existing matrix if they desired). My original comment was trying to say that it was correct.

Reply:
Alright, got it now... sorry.